pkg/cmd/admin/policy/reconcile_sccs_test.go

```diff
@@ -15,7 +15,7 @@ func TestComputeDefinitions(t *testing.T) {
 	diffCaps.AllowedCapabilities = []kapi.Capability{"foo"}
 
 	diffHostDir := goodSCC()
-	diffHostDir.AllowHostDirVolumePlugin = true
+	diffHostDir.Volumes = []kapi.FSType{kapi.FSTypeHostPath}
```

Comment:
internal no longer has the Allow field so it must be tested through the Volumes slice.